src/state/slices/opportunitiesSlice/selectors/lpSelectors.ts

```diff
@@ -193,15 +193,16 @@ export const selectAggregatedEarnUserLpOpportunities = createDeepEqualOutputSele
 
     if (!portfolioAssetBalancesById) return opportunities
 
-    for (const [lpId, opportunityMetadata] of Object.entries(lpOpportunitiesById)) {
+    for (const opportunityMetadata of Object.values(lpOpportunitiesById)) {
       if (!opportunityMetadata) continue
+      const underlyingAssetId = opportunityMetadata.underlyingAssetId
 
-      const lpAsset = assets[lpId as AssetId]
+      const lpAsset = assets[underlyingAssetId]
```

Comment:
`lpId` could be a composite asset, and it was for Portals, this never worked, but accidentally was happy in the default "By Provider", see description for context